runtime/syscall-server/syscall_server_utils.cpp

```diff
@@ -24,7 +25,7 @@
 #endif
 namespace bpftime
 {
-static bool already_setup = false;
+static std::once_flag g_startup_once;
```

Comment:
Does `std::once_flag` requires call a an initializer function? If so, will it work correctly if bpftime was loaded before initialization of global variables?